src/collections/blog/2024/11-05-what-is-the-kanvas-catalog/post.mdx

```diff
@@ -17,32 +17,114 @@ resource: true
+<Link to="/cloud-native-management/catalog">Kanvas Catalog</Link> is a hub for sharing
+and discovering best practices, reusable templates, and operational patterns for
+Kubernetes and cloud-native infrastructure. It's like a marketplace where you can
+find and contribute pre-built infrastructure configurations and operational views.
+The Catalog is a part of the Kanvas platform, which is a comprehensive suite of tools
+for managing cloud-native infrastructure.
 
-<div className="note"> <a href="https://meshery.layer5.io/catalog">Explore the catalog</a></div>
+<div className="note">
+  {" "}
+  <a href="https://meshery.layer5.io/catalog">Explore the catalog</a>
+</div>
 
 ### What can you find in the Catalog?
 
-- Design Patterns: Ready-made blueprints for common infrastructure and application architectures. These patterns can save you significant time and effort in designing your deployments.  
+- Design Patterns: Ready-made blueprints for common infrastructure and application architectures. These patterns can save you significant time and effort in designing your deployments.
 - Filters and Applications: Pre-configured filters for Envoy proxies, WebAssembly filters, and complete application deployments.
-- Meshery Designs: Share and reuse your own Meshery configurations, making it easier to collaborate and standardize your deployments.  
+- Meshery Designs: Share and reuse your own Meshery configurations, making it easier to collaborate and standardize your deployments.
 - Meshery Models: Share and reuse your own Meshery models, making it easier to collaborate and standardize your component library.
 
 #### Why is the Catalog useful?
 
 - Accelerated Development: Leverage existing patterns to jumpstart your projects and avoid reinventing the wheel.
-- Community Knowledge: Benefit from the collective experience of the Layer5 community and industry best practices.  
+- Community Knowledge: Benefit from the collective experience of the Layer5 community and industry best practices.
 - Standardization: Promote consistency and reduce errors by using predefined configurations.
 - Collaboration: Share your own designs and contribute to the growing collection of patterns.
 
 ### How can you contribute to the Catalog?
- 
 
+Contributing to the Meshery Catalog is a great way to share your cloud-native configurations with the community. By submitting your models and patterns, you not only showcase your work but also help others benefit from your innovations. This guide outlines how to use `mesheryctl` for uploading your contributions efficiently.
```

Comment:
Are we talking about the Meshery Catalog? 

Double check the title of this post.